opencti-platform/opencti-front/src/private/components/workspaces/investigations/InvestigationGraph.jsx

```diff
@@ -341,6 +341,22 @@ const investigationGraphStixSightingRelationshipQuery = graphql`
     }
 `;
 
+// To count the number of relationships for MetaObjects
+const investigationGraphStixMetaRelCountQuery = graphql`
```

Comment:
For meta objects we cannot use the same process as for core objects to compute the number of relationships.
So we had to use a specific query for those.